### PR TITLE
feat: remove the basic auth section of the ingress

### PR DESCRIPTION
**Pull request by bgroff**
Remove the basic auth section of the ingress and the function for creating the basic auth credentials. This will be handled in the application in the next release.

### CODE REVIEW

File: internal/cmd/local/local/cmd.go [hunk 1]
```diff
@@ -357,6 +356,7 @@ func (c *Command) Install(ctx context.Context, opts InstallOpts) error {
 		fmt.Sprintf("global.env_vars.AIRBYTE_INSTALLATION_ID=%s", telUser),
 		fmt.Sprintf("global.jobs.resources.limits.cpu=3"),
 		fmt.Sprintf("global.jobs.resources.limits.memory=4Gi"),
+		//fmt.Sprintf("global.auth.enabled=true"),
```
Comment (bgroff) [inline internal/cmd/local/local/cmd.go:359]: @colesnodgrass This should be the same as setting

```
global:
  auth
    enabled: true
```

right?

Comment (colesnodgrass) [reply internal/cmd/local/local/cmd.go:359]: Correct.
---
Unrelated to this question, but in looking at this code, there is to reason to use `fmt.Sprintf` here, or in the two lines above (which I originally wrote 😞)

Comment (bgroff) [reply internal/cmd/local/local/cmd.go]: No, I was just following your lead. A string would be fine.

File: internal/cmd/local/k8s/client.go [hunk 2]
```diff
@@ -206,6 +207,14 @@ func (d *DefaultK8sClient) SecretCreateOrUpdate(ctx context.Context, secretType
 	return fmt.Errorf("unexpected error while handling the secret %s: %w", name, err)
 }
 
+func (d *DefaultK8sClient) SecretGet(ctx context.Context, namespace, name string) (*corev1.Secret, error) {
+	secret, err := d.ClientSet.CoreV1().Secrets(namespace).Get(ctx, name, metav1.GetOptions{})
+	if err != nil {
+		return nil, fmt.Errorf("unable to update the secret %s: %w", name, err)
```
Comment (colesnodgrass) [inline internal/cmd/local/k8s/client.go:213]:
```suggestion
		return nil, fmt.Errorf("unable to get the secret %s: %w", name, err)
```

File: internal/cmd/local/local/cmd.go [hunk 3]
```diff
@@ -357,6 +356,7 @@ func (c *Command) Install(ctx context.Context, opts InstallOpts) error {
 		fmt.Sprintf("global.env_vars.AIRBYTE_INSTALLATION_ID=%s", telUser),
 		fmt.Sprintf("global.jobs.resources.limits.cpu=3"),
 		fmt.Sprintf("global.jobs.resources.limits.memory=4Gi"),
+		fmt.Sprintf("global.auth.enabled=true"),
```
Comment (colesnodgrass) [inline internal/cmd/local/local/cmd.go:359]: I'm going to remove all these extra `fmt.Sprintf` calls in a follow-up PR.

File: internal/cmd/local/local_install.go [hunk 4]
```diff
@@ -174,7 +174,7 @@ func NewCmdInstall(provider k8s.Provider) *cobra.Command {
 					return err
 				}
 
-				spinner.Success("Airbyte installation complete")
+				spinner.Success("Airbyte installation complete. Run: abctl local get-credentials to retrieve your credentials")
```
Comment (colesnodgrass) [inline internal/cmd/local/local_install.go:177]:
```suggestion
				spinner.Success("Airbyte installation complete.\nRun: abctl local get-credentials to retrieve your credentials")
```

File: internal/cmd/local/local_get_credentials.go [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+				client, err := defaultK8s(provider.Kubeconfig, provider.Context)
+				if err != nil {
+					return nil
+				}
+				secret, err := client.SecretGet(cmd.Context(), airbyteNamespace, airbyteAuthSecretName)
+				if err != nil {
+					return err
+				}
+
+				pterm.Success.Printfln(fmt.Sprintf("Getting your credentials: %s", secret.Name))
```
Comment (colesnodgrass) [inline internal/cmd/local/local_get_credentials.go:41]:
```suggestion
				pterm.Success.Println(fmt.Sprintf("Getting your credentials: %s", secret.Name))
```

File: internal/cmd/local/local_get_credentials.go [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+				for k, v := range secret.Data {
+					if k == "jwt-signature-secret" {
+						continue
+					}
```
Comment (colesnodgrass) [inline internal/cmd/local/local_get_credentials.go:45]: Rather then being explicit about what we don't output, should we rather be explicit about what we _do_ output?

File: internal/cmd/local/local_get_credentials.go [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+func NewCmdGetCredentials(provider k8s.Provider) *cobra.Command {
+	//spinner := &pterm.DefaultSpinner
+
+	cmd := &cobra.Command{
+		Use:   "get-credentials",
+		Short: "Get Airbyte user credentials",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return nil
+		},
```
Comment (colesnodgrass) [inline internal/cmd/local/local_get_credentials.go:27]: This can be removed

File: internal/cmd/local/local_get_credentials.go [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const (
+	airbyteAuthSecretName = "airbyte-auth-secrets"
+	airbyteNamespace      = "airbyte-abctl"
+)
+
+func NewCmdGetCredentials(provider k8s.Provider) *cobra.Command {
+	//spinner := &pterm.DefaultSpinner
```
Comment (colesnodgrass) [inline internal/cmd/local/local_get_credentials.go:20]:
```suggestion
```

**Review by colesnodgrass — APPROVED** [review]

File: internal/cmd/local/local_get_credentials.go [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+
+	cmd := &cobra.Command{
+		Use:   "get-credentials",
```
Comment (bgroff) [inline internal/cmd/local/local_get_credentials.go:23]: Can to just credentials

File: internal/cmd/local/local_get_credentials.go [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+		Short: "Get Airbyte user credentials",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return telClient.Wrap(cmd.Context(), telemetry.Install, func() error {
+
+				// TODO: We should check that we can actually talk to the cluster
+				client, err := defaultK8s(provider.Kubeconfig, provider.Context)
+				if err != nil {
```
Comment (bgroff) [inline internal/cmd/local/local_get_credentials.go:33]: This should print an error is we cannot connect to the Airbyte instance

**Pull request merged**